include/ddwaf.h

```diff
@@ -114,10 +114,22 @@ struct _ddwaf_object
  **/
 struct _ddwaf_config
 {
-    /** Maximum length of ddwaf::object arrays. */
-    uint64_t maxArrayLength;
-    /** Maximum depth of ddwaf::object maps. */
-    uint64_t maxMapDepth;
+    struct {
+        /** Maximum size of ddwaf::object containers. */
+        uint32_t max_container_size;
+        /** Maximum depth of ddwaf::object containers. */
+        uint32_t max_container_depth;
+        /** Maximum length of ddwaf::object strings. */
+        uint32_t max_string_length;
+    } limits;
+
+    /** Obfuscator regexes - the strings are owned by the caller */
```

Comment:
What about a simpler API by copying them instead and releasing them on ddwaf_destroy?

Reply:
I don't want to copy the strings because I don't need them after the regex has been created. This allows you to initialise the structure with literals.

Reply:
SGTM, as long as you are aware of this.